### PR TITLE
Add support for the 'mah capacity exceeded' osd warning

### PR DESCRIPTION
**Pull request by rvdveen**
Adds support for the warning added in betaflight/betaflight#9454

### CODE REVIEW

File: src/js/tabs/osd.js [hunk 1]
```diff
@@ -1626,6 +1631,7 @@ OSD.chooseFields = function () {
             F.RSSI,
             F.LINK_QUALITY,
             F.RSSI_DBM,
+            F.OVER_CAP,
```
Comment (McGiverGim) [inline src/js/tabs/osd.js]: If this is for Betaflight 4.2, it needs to be done only for MSP version 1.43.

File: src/js/tabs/osd.js [hunk 2]
```diff
@@ -1332,6 +1332,11 @@ OSD.constants = {
             text: 'osdWarningTextRssiDbm',
             desc: 'osdWarningRssiDbm'
         },
+        OVER_CAP: {
+            name: 'OVER_CAP',
+            text: 'osdWarningTextOverCap',
+            desc: 'osdWarningOverCap'
```
Comment (McGiverGim) [inline src/js/tabs/osd.js]: As Sonar issue, add a trailing comma to the latest element.

Comment (rvdveen) [reply src/js/tabs/osd.js]: Will do, thought it'd be inconsistent with the other statements above it that don't have the trailing comma

File: src/js/tabs/osd.js (hunk 1 above)
Comment (rvdveen) [reply src/js/tabs/osd.js]: Ah I wasn't sure about that, it didn't seem visible on a quad that didn't have support for the warning.

File: src/js/tabs/osd.js (hunk 2 above)
Comment (McGiverGim) [reply src/js/tabs/osd.js]: Yes, we are trying to unify the code, so the new code must be according to our Sonar rules. Maybe in a future PR we fix the rest of Sonar issues of this file ;)

File: locales/en/messages.json [hunk 3]
```diff
@@ -5020,6 +5020,13 @@
     "osdWarningRssiDbm": {
         "message": "Warns when RSSI dBm value is below alarm setting"
     },
+    "osdWarningTextOverCap": {
+        "message": "mAh capacity exceeded",
+        "description": "One of the warnings that can be selected to be shown in the OSD"
+    },
```
Comment (McGiverGim) [inline locales/en/messages.json]: One thing more, to discuss... now the OSD elements (including the warnings) are ordered by alphabetical order. I think is better to include this warning near the other battery warnings. Maybe with something like:
- "Battery mAh capacity exceeded" or maybe "Battery: mAh capacity exceeded"

What do you think?

Comment (rvdveen) [reply locales/en/messages.json]: Might be clearer but it's a bit too long compared to the other warning labels. How about "Battery over capacity"? Not too long and still descriptive. If you prefer "Battery mAh capacity exceeded" despite the length I'll add that

Comment (McGiverGim) [reply locales/en/messages.json]: Let's wait for other thoughts... @mikeller what do you think?

Comment (mikeller) [reply locales/en/messages.json]: 'Battery capacity exceeded' or 'Battery over capacity' sounds about right - the units used to measure this are not really relevant anyway.

Comment (rvdveen) [reply locales/en/messages.json]: "Battery over capacity" it is 👍

**Pull request merged**